examples/pipelines/providers/anthropic_manifold_pipeline.py

```diff
@@ -139,38 +166,107 @@ def pipe(
             }
 
             if body.get("stream", False):
+                supports_thinking = "claude-3-7" in model_id
+                reasoning_effort = body.get("reasoning_effort", "none")
+                budget_tokens = REASONING_EFFORT_BUDGET_TOKEN_MAP.get(reasoning_effort)
+
+                # Allow users to input an integer value representing budget tokens
+                if (
+                    not budget_tokens
+                    and reasoning_effort not in REASONING_EFFORT_BUDGET_TOKEN_MAP.keys()
+                ):
+                    try:
+                        budget_tokens = int(reasoning_effort)
+                    except ValueError as e:
```

Comment:
```suggestion
                                except (ValueError, TypeError) as e:
```

Reply:
Reason for that is that reasoning_effort can be of NoneType and then this fails with:

```
ERROR [open_webui.functions] Error: int() argument must be a string, a bytes-like object or a real number, not 'NoneType'
```

Could also do a check for reasoning_effort to not be NoneType. (No idea why body.get() in that case would not do the default value)